### DISCUSSION TITLE
Unable to drag regions on mobile with v7

### DISCUSSION THREAD

**Discussion by erik-iglikov** [katspaugh/wavesurfer.js#2816] [General]
Hey [katspaugh](https://github.com/katspaugh), first of all thank you and the folks who help for such a great library, love the work you do!

I'm trying to implement draggable regions on mobile. It works for the desktop but doesn't allow me to drag the region on mobile or mobile simulator in Chrome. I've tested [regions example from docs] (https://wavesurfer-js.org/examples/#regions.js) and didn't work there either (on mobile).

My setting for RegionPlugin:
`
const DEFAULT_REGION_CONFIG: RegionParams = {
  id: 'default',
  start: 0,
  end: 30,
  resize: false,
  drag: true,
};
`

Please let me know your thoughts on it.

**Comment by katspaugh** [comment]
True, good catch.
